### PR TITLE
Add missing bracket in glossary

### PR DESCRIPTION
**Pull request by palango**
Add missing bracket in glossary

### CODE REVIEW

**Comment by LefterisJP** [comment]
Α hint: You should make commits that should skip the tests by adding `[skip ci]` in the commit title.

Looks good to me.

**Comment by LefterisJP** [comment]
Ah wait no!

File: docs/glossary.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ Raiden Glossary
        Balance proof is any kind of message we use in order to cryptographically prove to our counterparty (or them to us) that their balance has changed and that we have received a transfer. 
 
    hashlock
-       A hashlock is the hashed secret that accompanis a locked message: ``sha3(secret``
```
Comment (LefterisJP) [inline docs/glossary.rst]: accompanis -> accompanies

**Comment by palango** [comment]
It should work everywhere: https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build

I'll fix the typo.

**Comment by LefterisJP** [comment]
> It should work everywhere: https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build

Yeah it should. Apologies, did not notice you actually put it in the commit message body as only the title shows. Please try to put it in the title so it's easily visible in github.

**Pull request merged**